meetings/2021-07-21.md

```diff
@@ -0,0 +1,176 @@
+But the best way is to have something in the field and get it used in the field.
+We need value from real; "in the field"; implementations and use.
+We need to see value from there.
+That's an essential part of maturing specs and getting them to completion to have something out there that is close to completion.
+I wanna ideally try to get there.
+
+Henry: That's what I was doing 7 years ago in a startup I was with. I had built a server and clients . But the startup did not work out. Then it took a long time because something got stuck at MIT.
+
+If you look at this UC3 PR, then you can see that WAC+ import solves a lot of issues.
+And it might help solve things on the ACP side too.
+It helps you avoid copying lots of data around.
+And answers many questions, such as "If you change the root ACR do changes get effected everywhere else?...
+
+Matthieu: I agree with Henry. I think it is problematic that some fundamental issues have not been formulated clearly in the past and that the fact that we're doing it proves that we're doing something right.
+
+Justin: Yes. More than anything, I just want to understand how do we optimize what we have left to do?
+How do we get back from implementation?
+
+Henry: Maybe we need 1 or 1.5 days a week for Matthieu and I to move along.
+These UC cases are very much like an implementation. It is backed by experience.
+I agree we need to move faster on this.
+
+Matthieu: I agree with Henry's assessment that we need more time aside mostly. There was also a problem of fluency and getting up to speed with the entire Solid ecosystem and state of the discussion on my side as well. But with my current understanding of WAC and Solid, we seem to move along much faster. I also agree that we want to get to implementation feedback as early as possible, but that the use cases are adequately filling a bit of that space for now.
+
+Justin: Absolutely.
+
+Pavlik: Yes. Another part of my suggestion is not expecting someone else from commenting on your PRs to make progress.
+
+Matthieu: Yes, you're perfectly right, that has been problematic in the past, I'll try to adjust to make faster progress and focus on important issues and questions during the panel.
+
+Henry: Adjusting expectations of comments to move along faster sounds good. We'll keep it in check so that it doesn't get out of control.
+Let's not lose the benefit of questions and feedback.
+
+Matthieu: +1
+
+### Closing outdated UCR issues
+
+Justin: +1
+
+Henry: I'll follow up on it on GitHub
+
+### Transfer of issue 191
+
+Justin: I might have permission to do it.
+
+Matthieu: I'll ask Sarven otherwise.
+
+
+## Actions
+
+* Matthieu to submit UC0 on effective access control resource discovery.
+* Matthieu to create issues:
+    * How does an agent know all the resources it has access to?
+    * How are required credentials to present to access a resource advertised?
+    * Using extending access modes instead of to many apply policy predicates in ACP.
```

Comment:
This doesn't seem clear to me. Might need some rewording.
```suggestion
    * Using extending access modes instead of too many apply policy predicates in ACP.
```